Mathlib/LinearAlgebra/LinearProgramming.lean

```diff
@@ -0,0 +1,83 @@
+/-
+Copyright (c) 2023 Martin Dvorak. All rights reserved.
+Released under Apache 2.0 license as described in the file LICENSE.
+Authors: Martin Dvorak
+-/
+import Mathlib.LinearAlgebra.Matrix.ToLin
+import Mathlib.LinearAlgebra.AffineSpace.AffineMap
+
+/-! # Linear programming
+
+Minimizing a linear function on a region defined by linear inequalities.
```

Comment:
This should include a discussion about the meaning of  "linear" in linear program vs the use of `linear` in mathlib, and why it's actually `affine`.